Lib/test/test_builtin.py

```diff
@@ -2190,6 +2190,18 @@ def __del__(self):
         self.assertEqual(["before", "after"], out.decode().splitlines())
 
 
+class ImmortalTests(unittest.TestCase):
+    def test_immortal(self):
+        none = None
+        true = True
+        false = False
+        small_int = 100
+        self.assertGreater(sys.getrefcount(none), 1e8)
+        self.assertGreater(sys.getrefcount(true), 1e8)
+        self.assertGreater(sys.getrefcount(false), 1e8)
+        self.assertGreater(sys.getrefcount(small_int), 1e8)
```

Comment:
How about also adding another ref for each and then checking to make sure it did not change?

Reply:
Yeah that sounds good!